lms/djangoapps/mobile_api/social_facebook/groups/views.py

```diff
@@ -0,0 +1,123 @@
+"""
+Views for groups info API
+"""
+
+from rest_framework import generics, status, mixins
+from rest_framework.response import Response
+from ...utils import mobile_view
+from lms.djangoapps.mobile_api.social_facebook.groups import serializers
+import facebook
+
+from django.conf import settings
+_FACEBOOK_API_VERSION = settings.FACEBOOK_API_VERSION
+_FACEBOOK_APP_ID = settings.FACEBOOK_APP_ID
+_FACEBOOK_APP_SECRET = settings.FACEBOOK_APP_SECRET
+
+@mobile_view()
+class Groups(generics.CreateAPIView, mixins.DestroyModelMixin):
+    """
+    **Use Case**
+
+        An API to Create or Delete course groups.
+
+    **Creation Example request**:
+
+        POST /api/mobile/v0.5/social/facebook/groups/<group_id>
```

Comment:
Best practices wise, it's pretty simple:
The `create` endpoint has the same path as the `delete` and `update` endpoints.  However, the `create` one just doesn't have the identifier included.

Here's a useful [API guidebook](http://pages.apigee.com/rs/apigee/images/api-design-ebook-2012-03.pdf) form Apigee that we're adapting at edX (with some of our own modifications).  The top of page 7 gives a summary overview of how to design the API paths.